File: core/src/main/java/com/lsxiao/rig/core/Validator.java
package com.lsxiao.rig.core;

import com.lsxiao.rig.core.ValidateResult;

/**
 * write with RigRule
 * author:lsxiao
 * date:2017-05-25 21:55
 * github:https://github.com/lsxiao
 * zhihu:https://zhihu.com/people/lsxiao
 */

public interface Validator {
    ValidateResult check(Object o);
}

File: core/src/main/java/com/lsxiao/rig/core/Validator.kt
package com.lsxiao.rig.core

/**
 * write with Checkable
 * author:lsxiao
 * date:2017-05-25 21:55
 * github:https://github.com/lsxiao
 * zhihu:https://zhihu.com/people/lsxiao
 */

interface Validator {
    fun check(o: Any, group: Int = 0): ValidateResult
}
